sys-kernel/dracut/files/gentoo-network-r1.patch

```diff
@@ -0,0 +1,22 @@
+commit e2863bfaaecdf5ff5251f545fdfac45dc6c75887
+Author: Laszlo Gombos <laszlo.gombos@gmail.com>
+Date:   Fri Dec 23 18:09:50 2022 +0000
```

Comment:
Please use `git format-patch` to generate a patch file with the right formatting.